music21/chord/__init__.py

```diff
@@ -1914,6 +1917,8 @@ def inversion(self, newInversion=None, *, find=True, testRoot=None, transposeOnS
 
                 if 'inversion' in self._overrides:
                     del self._overrides['inversion']
+                if 'bass' in self._overrides:
+                    del self._overrides['bass']
```

Comment:
ah! I thought we were getting rid of bass overrides?

Reply:
Still being set here: https://github.com/cuthbertLab/music21/blob/d3770d0907e3af00d45153da25b8c064bf2bbec4/music21/harmony.py#L210-L214

Was there an effort to get rid of this before?

Reply:
I just remember something about, "no one overrides the bass note!  just the roots.  But yes, if it's still being set somewhere then we should keep it going.

Reply:
AH, got it, when we were trying to paraphrase Rameau better.

Reply:
Ah -- bass can also be overridden to a different octave.